src/form-builder.tsx

```diff
@@ -13,7 +13,7 @@ import {
 import { Container, Wrapper } from "./form-builder.styles";
 import { Translator } from "./translator/translator";
 
-interface IPrefillSchema {
+export interface IPrefillSchema {
     [key: string]: IPrefillAttributes | IPrefillAttributes[];
 }
```

Comment:
since this interface is used by the translator only, move this to a new `translator/types.ts` and rename it `IPrefillConfig`